.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,21 @@
+// .devcontainer/devcontainer.json
+{
+  "name": "Embedded Sharepoint",
+  "dockerFile": "../dockerfile",
+  "context": "..",
```

Comment:
You need to configure this file such that it does the following:
1. Builds image if it doesn't exist yet and then load image 
2. Launch new container from image
3. Pull project into container using context that you've already done. 

Not perfectly sure (writing on phone), but if you add this, I believe it will spin up the container on demand properly.

```
"build": {
		"context": "..", 
		"dockerfile": "../Dockerfile"
}
```

Reduce as much dependency on bash script as possible. Bash scripts are so flexible that it becomes hard to prevent creep. Do as much with the limited tool as possible.

Reply:
.devcontainer + VS Code extension accomplishes this
Bash script replicates this for CLI users

Resolved 8eb6ea9e2a8cf2297886c0cb72ef63b4318e90a9